source/gui/settingsDialogs.py

```diff
@@ -3204,34 +3204,35 @@ class AddonStorePanel(SettingsPanel):
 
 	def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 		sHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
-		# Translators: This is a label for the automatic updates combo box in the Add-on Store Settings dialog.
-		automaticUpdatesLabelText = _("Automatic &updates:")
-		self.automaticUpdatesComboBox = sHelper.addLabeledControl(
-			automaticUpdatesLabelText,
+		# Translators: This is a label for the add-ons update combo box in the Add-on Store Settings dialog.
+		addonsUpdateLabelText = _("Add-ons &update:")
```

Comment:
This feels ungrammatical to me. I would prefer "Add-on updates".
Notwithstanding this, I think if we want to change the label, we need to rename "Disabled" to "Manual", as "Disabled" seems to imply that updating add-ons is entirely disabled (i.e. add-ons are locked at the currently installed version).

Reply:
I agree with Sascha here with this change being misleading.

```suggestion
		addonsUpdateLabelText = _("Add-on &update check:")
```